drunc_docker_service/Dockerfile

```
@@ -13,13 +13,12 @@ WORKDIR /basedir
 ENV SPACK_ROOT /cvmfs/dunedaq.opensciencegrid.org/spack/releases/fddaq-v5.1.0-a9-1/spack-0.20.0
```

This needs to change because we're no longer using `fddaq-v5.1.0-a9-1`

Locally I found that this path now works:
```suggestion
ENV SPACK_ROOT /cvmfs/dunedaq-development.opensciencegrid.org/nightly/NFD_DEV_241114_A9/spack-0.22.0
```

---

How did you happen to find that new path????